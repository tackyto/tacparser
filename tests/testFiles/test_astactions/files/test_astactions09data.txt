"Test01" {
    "FamilyName01" {
        {name = "Aaaa Aaaa", age = 41, sex = Man, phone = 0123-4567-8901 }
        {name = "Aaaa Bbbb", age = 42, sex = Female, phone = 0123-4567-8902 }
        {name = "Aaaa Cccc", age = 15, sex = M}
        {name = "Aaaa Dddd", age = 12, sex = F}
    }
    "FamilyName02" {
        {name = "Bbbb Aaaa", age = 45, sex = Man, phone = 0123-4567-8901 }
        {name = "Bbbb Bbbb", age = 44, sex = Female }
        {name = "Bbbb Cccc", age = 13, sex = M}
        {name = "Bbbb Dddd", age = 11}
    }
    "FamilyName03" {
        {name = "Cccc Aaaa", age = 45, sex = Man, phone = 0123-4567-8903 }
        {name = "Cccc Bbbb", age = 44, sex = Female }
    }
    "FamilyName04" {
        {name = "Aaaa Aaaa", age = 81, sex = M, phone = 0123-4567-8904 }
    }
    "FamilyName05" {
        {name = "Dddd Aaaa", age = 43, sex = Man, phone = 0123-4567-8905-1111 }
    }
}
