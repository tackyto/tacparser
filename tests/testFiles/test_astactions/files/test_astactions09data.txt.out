ASTActionsTest : (1, 0 - 24, 2 : 0 - 895) : {}
    Society : (1, 0 - 24, 2 : 0 - 895) : {}
        SocietyName : (1, 0 - 1, 9 : 0 - 9) : {}
            Literal : (1, 0 - 1, 9 : 0 - 9) : {}
                @Tarminal : (1, 1 - 1, 7 : 1 - 7) "Test01"
        Family : (2, 4 - 8, 4 : 15 - 291) : {}
            FamilyName : (2, 4 - 2, 19 : 15 - 30) : {}
                Literal : (2, 4 - 2, 19 : 15 - 30) : {}
                    @Tarminal : (2, 5 - 2, 17 : 16 - 28) "FamilyName01"
            Person : (3, 9 - 3, 73 : 41 - 105) : {name: Aaaa Aaaa}
                Name : (3, 9 - 3, 29 : 41 - 61) : {}
                    Literal : (3, 16 - 3, 27 : 48 - 59) : {}
                        @Tarminal : (3, 17 - 3, 26 : 49 - 58) "Aaaa Aaaa"
                Age : (3, 29 - 3, 39 : 61 - 71) : {}
                    Number : (3, 35 - 3, 37 : 67 - 69) : {}
                        @Tarminal : (3, 35 - 3, 37 : 67 - 69) "41"
                Sex : (3, 39 - 3, 50 : 71 - 82) : {}
                    MorF : (3, 45 - 3, 48 : 77 - 80) : {}
                        Male : (3, 45 - 3, 48 : 77 - 80) : {}
                Phone : (3, 50 - 3, 73 : 82 - 105) : {}
                    NumberHyphen : (3, 58 - 3, 73 : 90 - 105) : {}
                        @Tarminal : (3, 58 - 3, 63 : 90 - 95) "0123-"
                        @Tarminal : (3, 63 - 3, 68 : 95 - 100) "4567-"
                        @Tarminal : (3, 68 - 3, 72 : 100 - 104) "8901"
            Person : (4, 9 - 4, 76 : 116 - 183) : {name: Aaaa Bbbb}
                Name : (4, 9 - 4, 29 : 116 - 136) : {}
                    Literal : (4, 16 - 4, 27 : 123 - 134) : {}
                        @Tarminal : (4, 17 - 4, 26 : 124 - 133) "Aaaa Bbbb"
                Age : (4, 29 - 4, 39 : 136 - 146) : {}
                    Number : (4, 35 - 4, 37 : 142 - 144) : {}
                        @Tarminal : (4, 35 - 4, 37 : 142 - 144) "42"
                Sex : (4, 39 - 4, 53 : 146 - 160) : {}
                    MorF : (4, 45 - 4, 51 : 152 - 158) : {}
                        Female : (4, 45 - 4, 51 : 152 - 158) : {}
                Phone : (4, 53 - 4, 76 : 160 - 183) : {}
                    NumberHyphen : (4, 61 - 4, 76 : 168 - 183) : {}
                        @Tarminal : (4, 61 - 4, 66 : 168 - 173) "0123-"
                        @Tarminal : (4, 66 - 4, 71 : 173 - 178) "4567-"
                        @Tarminal : (4, 71 - 4, 75 : 178 - 182) "8902"
            Person : (5, 9 - 5, 46 : 194 - 231) : {name: Aaaa Cccc}
                Name : (5, 9 - 5, 29 : 194 - 214) : {}
                    Literal : (5, 16 - 5, 27 : 201 - 212) : {}
                        @Tarminal : (5, 17 - 5, 26 : 202 - 211) "Aaaa Cccc"
                Age : (5, 29 - 5, 39 : 214 - 224) : {}
                    Number : (5, 35 - 5, 37 : 220 - 222) : {}
                        @Tarminal : (5, 35 - 5, 37 : 220 - 222) "15"
                Sex : (5, 39 - 5, 46 : 224 - 231) : {}
                    MorF : (5, 45 - 5, 46 : 230 - 231) : {}
                        Male : (5, 45 - 5, 46 : 230 - 231) : {}
            Person : (6, 9 - 6, 46 : 242 - 279) : {name: Aaaa Dddd}
                Name : (6, 9 - 6, 29 : 242 - 262) : {}
                    Literal : (6, 16 - 6, 27 : 249 - 260) : {}
                        @Tarminal : (6, 17 - 6, 26 : 250 - 259) "Aaaa Dddd"
                Age : (6, 29 - 6, 39 : 262 - 272) : {}
                    Number : (6, 35 - 6, 37 : 268 - 270) : {}
                        @Tarminal : (6, 35 - 6, 37 : 268 - 270) "12"
                Sex : (6, 39 - 6, 46 : 272 - 279) : {}
                    MorF : (6, 45 - 6, 46 : 278 - 279) : {}
                        Female : (6, 45 - 6, 46 : 278 - 279) : {}
        Family : (8, 4 - 14, 4 : 291 - 534) : {}
            FamilyName : (8, 4 - 8, 19 : 291 - 306) : {}
                Literal : (8, 4 - 8, 19 : 291 - 306) : {}
                    @Tarminal : (8, 5 - 8, 17 : 292 - 304) "FamilyName02"
            Person : (9, 9 - 9, 73 : 317 - 381) : {name: Bbbb Aaaa}
                Name : (9, 9 - 9, 29 : 317 - 337) : {}
                    Literal : (9, 16 - 9, 27 : 324 - 335) : {}
                        @Tarminal : (9, 17 - 9, 26 : 325 - 334) "Bbbb Aaaa"
                Age : (9, 29 - 9, 39 : 337 - 347) : {}
                    Number : (9, 35 - 9, 37 : 343 - 345) : {}
                        @Tarminal : (9, 35 - 9, 37 : 343 - 345) "45"
                Sex : (9, 39 - 9, 50 : 347 - 358) : {}
                    MorF : (9, 45 - 9, 48 : 353 - 356) : {}
                        Male : (9, 45 - 9, 48 : 353 - 356) : {}
                Phone : (9, 50 - 9, 73 : 358 - 381) : {}
                    NumberHyphen : (9, 58 - 9, 73 : 366 - 381) : {}
                        @Tarminal : (9, 58 - 9, 63 : 366 - 371) "0123-"
                        @Tarminal : (9, 63 - 9, 68 : 371 - 376) "4567-"
                        @Tarminal : (9, 68 - 9, 72 : 376 - 380) "8901"
            Person : (10, 9 - 10, 52 : 392 - 435) : {name: Bbbb Bbbb}
                Name : (10, 9 - 10, 29 : 392 - 412) : {}
                    Literal : (10, 16 - 10, 27 : 399 - 410) : {}
                        @Tarminal : (10, 17 - 10, 26 : 400 - 409) "Bbbb Bbbb"
                Age : (10, 29 - 10, 39 : 412 - 422) : {}
                    Number : (10, 35 - 10, 37 : 418 - 420) : {}
                        @Tarminal : (10, 35 - 10, 37 : 418 - 420) "44"
                Sex : (10, 39 - 10, 52 : 422 - 435) : {}
                    MorF : (10, 45 - 10, 52 : 428 - 435) : {}
                        Female : (10, 45 - 10, 52 : 428 - 435) : {}
            Person : (11, 9 - 11, 46 : 446 - 483) : {name: Bbbb Cccc}
                Name : (11, 9 - 11, 29 : 446 - 466) : {}
                    Literal : (11, 16 - 11, 27 : 453 - 464) : {}
                        @Tarminal : (11, 17 - 11, 26 : 454 - 463) "Bbbb Cccc"
                Age : (11, 29 - 11, 39 : 466 - 476) : {}
                    Number : (11, 35 - 11, 37 : 472 - 474) : {}
                        @Tarminal : (11, 35 - 11, 37 : 472 - 474) "13"
                Sex : (11, 39 - 11, 46 : 476 - 483) : {}
                    MorF : (11, 45 - 11, 46 : 482 - 483) : {}
                        Male : (11, 45 - 11, 46 : 482 - 483) : {}
            Person : (12, 9 - 12, 37 : 494 - 522) : {name: Bbbb Dddd}
                Name : (12, 9 - 12, 29 : 494 - 514) : {}
                    Literal : (12, 16 - 12, 27 : 501 - 512) : {}
                        @Tarminal : (12, 17 - 12, 26 : 502 - 511) "Bbbb Dddd"
                Age : (12, 29 - 12, 37 : 514 - 522) : {}
                    Number : (12, 35 - 12, 37 : 520 - 522) : {}
                        @Tarminal : (12, 35 - 12, 37 : 520 - 522) "11"
        Family : (14, 4 - 18, 4 : 534 - 690) : {}
            FamilyName : (14, 4 - 14, 19 : 534 - 549) : {}
                Literal : (14, 4 - 14, 19 : 534 - 549) : {}
                    @Tarminal : (14, 5 - 14, 17 : 535 - 547) "FamilyName03"
            Person : (15, 9 - 15, 73 : 560 - 624) : {name: Cccc Aaaa}
                Name : (15, 9 - 15, 29 : 560 - 580) : {}
                    Literal : (15, 16 - 15, 27 : 567 - 578) : {}
                        @Tarminal : (15, 17 - 15, 26 : 568 - 577) "Cccc Aaaa"
                Age : (15, 29 - 15, 39 : 580 - 590) : {}
                    Number : (15, 35 - 15, 37 : 586 - 588) : {}
                        @Tarminal : (15, 35 - 15, 37 : 586 - 588) "45"
                Sex : (15, 39 - 15, 50 : 590 - 601) : {}
                    MorF : (15, 45 - 15, 48 : 596 - 599) : {}
                        Male : (15, 45 - 15, 48 : 596 - 599) : {}
                Phone : (15, 50 - 15, 73 : 601 - 624) : {}
                    NumberHyphen : (15, 58 - 15, 73 : 609 - 624) : {}
                        @Tarminal : (15, 58 - 15, 63 : 609 - 614) "0123-"
                        @Tarminal : (15, 63 - 15, 68 : 614 - 619) "4567-"
                        @Tarminal : (15, 68 - 15, 72 : 619 - 623) "8903"
            Person : (16, 9 - 16, 52 : 635 - 678) : {name: Cccc Bbbb}
                Name : (16, 9 - 16, 29 : 635 - 655) : {}
                    Literal : (16, 16 - 16, 27 : 642 - 653) : {}
                        @Tarminal : (16, 17 - 16, 26 : 643 - 652) "Cccc Bbbb"
                Age : (16, 29 - 16, 39 : 655 - 665) : {}
                    Number : (16, 35 - 16, 37 : 661 - 663) : {}
                        @Tarminal : (16, 35 - 16, 37 : 661 - 663) "44"
                Sex : (16, 39 - 16, 52 : 665 - 678) : {}
                    MorF : (16, 45 - 16, 52 : 671 - 678) : {}
                        Female : (16, 45 - 16, 52 : 671 - 678) : {}
        Family : (18, 4 - 21, 4 : 690 - 790) : {}
            FamilyName : (18, 4 - 18, 19 : 690 - 705) : {}
                Literal : (18, 4 - 18, 19 : 690 - 705) : {}
                    @Tarminal : (18, 5 - 18, 17 : 691 - 703) "FamilyName04"
            Person : (19, 9 - 19, 71 : 716 - 778) : {name: Aaaa Aaaa}
                Name : (19, 9 - 19, 29 : 716 - 736) : {}
                    Literal : (19, 16 - 19, 27 : 723 - 734) : {}
                        @Tarminal : (19, 17 - 19, 26 : 724 - 733) "Aaaa Aaaa"
                Age : (19, 29 - 19, 39 : 736 - 746) : {}
                    Number : (19, 35 - 19, 37 : 742 - 744) : {}
                        @Tarminal : (19, 35 - 19, 37 : 742 - 744) "81"
                Sex : (19, 39 - 19, 48 : 746 - 755) : {}
                    MorF : (19, 45 - 19, 46 : 752 - 753) : {}
                        Male : (19, 45 - 19, 46 : 752 - 753) : {}
                Phone : (19, 48 - 19, 71 : 755 - 778) : {}
                    NumberHyphen : (19, 56 - 19, 71 : 763 - 778) : {}
                        @Tarminal : (19, 56 - 19, 61 : 763 - 768) "0123-"
                        @Tarminal : (19, 61 - 19, 66 : 768 - 773) "4567-"
                        @Tarminal : (19, 66 - 19, 70 : 773 - 777) "8904"
        Family : (21, 4 - 24, 0 : 790 - 893) : {}
            FamilyName : (21, 4 - 21, 19 : 790 - 805) : {}
                Literal : (21, 4 - 21, 19 : 790 - 805) : {}
                    @Tarminal : (21, 5 - 21, 17 : 791 - 803) "FamilyName05"
            Person : (22, 9 - 22, 78 : 816 - 885) : {name: Dddd Aaaa}
                Name : (22, 9 - 22, 29 : 816 - 836) : {}
                    Literal : (22, 16 - 22, 27 : 823 - 834) : {}
                        @Tarminal : (22, 17 - 22, 26 : 824 - 833) "Dddd Aaaa"
                Age : (22, 29 - 22, 39 : 836 - 846) : {}
                    Number : (22, 35 - 22, 37 : 842 - 844) : {}
                        @Tarminal : (22, 35 - 22, 37 : 842 - 844) "43"
                Sex : (22, 39 - 22, 50 : 846 - 857) : {}
                    MorF : (22, 45 - 22, 48 : 852 - 855) : {}
                        Male : (22, 45 - 22, 48 : 852 - 855) : {}
                Phone : (22, 50 - 22, 78 : 857 - 885) : {}
                    NumberHyphen : (22, 58 - 22, 78 : 865 - 885) : {}
                        @Tarminal : (22, 58 - 22, 63 : 865 - 870) "0123-"
                        @Tarminal : (22, 63 - 22, 68 : 870 - 875) "4567-"
                        @Tarminal : (22, 68 - 22, 73 : 875 - 880) "8905-"
                        @Tarminal : (22, 73 - 22, 77 : 880 - 884) "1111"
    _EOF : (24, 2 - 24, 2 : 895 - 895) : {}
